tfx/utils/kube_utils.py

```diff
@@ -132,13 +135,67 @@ def MakeCoreV1Api(self) -> k8s_client.CoreV1Api:  # pylint: disable=invalid-name
       self._LoadConfig()
     return k8s_client.CoreV1Api()
 
+  def MakeBatchV1Api(self) -> k8s_client.BatchV1Api:  # pylint: disable=invalid-name
+    """Make a kubernetes BatchV1Api client."""
+    if not self._config_loaded:
+      self._LoadConfig()
+    return k8s_client.BatchV1Api()
+
 _factory = _KubernetesClientFactory()
 
+def _sanitize_pod_name(pod_name: Text) -> Text:
```

Comment:
Your PR adds this in two places...can we consolidate?

Reply:
Yes, made this a public method.